examples/gno.land/p/samourai/daocond/daocond.gno

```diff
@@ -0,0 +1,26 @@
+package daocond
+
+import (
+	"gno.land/p/demo/json"
+)
+
+// We made conditions stateless to avoid complexity of handling events like (member added, member removed, role assigned, role removed, etc.)
+// This model should work pretty good for small DAOs with few voters.
+// In future, we should probably add a stateful model for large DAOs with many voters.
+
+type Condition interface {
+	Eval(votes map[string]Vote) bool
+	Signal(votes map[string]Vote) float64
+
+	Render() string
+	RenderJSON() *json.Node
+	RenderJSONWithVotes(votes map[string]Vote) *json.Node
```

Comment:
```suggestion
	Render() string
	RenderWithVotes(votes map[string]Vote) string
```

the json encoders should be a separate interface so devs can choose to not implement it

Reply:
[f0cfcf9](https://github.com/samouraiworld/gno/pull/3/commits/f0cfcf9b68ab12348aea143ee07b58b8b891eb46)
its on the other PR, i'll merge on this after